repository-data/development/src/main/resources/hcm-config/main.yaml

```diff
@@ -2,7 +2,6 @@
 definitions:
   config:
     /hippo:configuration/hippo:modules/autoexport/hippo:moduleconfig:
-      autoexport:enabled: false
```

Comment:
Just checking: is this disabled because we also have this clause in the `main.yaml` in module `local`?

Reply:
Spot on